kotsadm/web/src/components/gitops/GitOpsDeploymentManager.jsx

```diff
@@ -113,21 +104,59 @@ class GitOpsDeploymentManager extends React.Component {
     }
   }
 
+  getGitops = async () => {
+    try {
+      const res = await fetch(`${window.env.API_ENDPOINT}/gitops/get`, {
+        headers: {
+          "Authorization": Utilities.getToken(),
+          "Content-Type": "application/json",
+        },
+        method: "GET",
+      });
+      if (!res.ok) {
+        if (res.status === 401) {
+          Utilities.logoutUser();
+          return;
+        }
+        console.log("failed to get gitops settings, unexpected status code", res.status);
+        return;
+      }
+
+      const freshGitops = await res.json()
+
+      if (freshGitops?.enabled) {
+        const selectedService = find(SERVICES, service => service.value === freshGitops.provider);
+        this.setState({
+          selectedService: selectedService ? selectedService : this.state.selectedService,
+          hostname: freshGitops.hostname || "",
+          gitops: freshGitops
+        })
+      } else {
+        this.setState({
+          gitops: freshGitops
+        })
+      }
+
+      return freshGitops
+    } catch(err) {
+      console.log(err);
+      throw err;
```

Comment:
probably shouldn’t throw error on catch here

Reply:
copied both of these from getAppsList above

Reply:
hmmm, might not as big of a problem as i think. can keep it 👍